addChain.md

```diff
@@ -15,13 +15,13 @@ Please keep chains in alphabetical order.
 
 `github-repo` -> The repo name of the location of the chain binary.
 
-`language` -> Used in the Docker `FROM` argument to create a a docker base image. OPTIONS: "go, rust, nix, imported". Use "imported" if you are not able to build the chain binary from source and are importing a pre-made docker container.
+`dockerfile` -> Which dockerfile strategy to use (folder names under dockerfile/). OPTIONS: `cosmos`, `cargo`, `imported`, or `none`. Use `imported` if you are importing an existing public docker image as a base for the heighliner image. Use `none` if you are not able to build the chain binary from source and need to download binaries into the image instead.
```

Comment:
What's the purpose and differences between `Dockerfile`, `native.Dockerfile`, `local.Dockerfile`? Is there any special handling to keep the 2 in sync?

Reply:
- `Dockerfile` is for buildkit. it is not compatible with standard docker build due to `FROM --platform=$BUILDPLATFORM` which is necessary in buildkit builds to cross compile from an image on the machine's arch.
- `native.Dockerfile` is for non-buildkit builds on your local machine, without the `-b` flag. It will fetch the chain source from the remote git repository similar to the `Dockerfile`
- `local.Dockerfile` is used when the `--local` flag is passed. This will use the current working directory for the chain source code and is mainly used for chain development flows as it does not require changes to be committed and pushed to the remote git repository to build

A contributor doc would be helpful to explain these differences to keep them in sync, as most changes should be made to all 3.

Reply:
I think all of that in the README would be excellent.